modules/commons/src/main/resources/schemas/neurosciencegraph/commons/selectors/v0.2.0.json

```diff
@@ -0,0 +1,176 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "SelectorShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "property": [
+        {
+          "path": "rdf:type",
+          "name": "Selector type",
+          "description": "The class of the Selector.",
+          "minCount": 1,
+          "maxCount": 1
+        }
```

Comment:
What is the purpose of this property shape "Selector type"? Aren't you enforcing the type anyways through the "class" key in the individual selector shapes?

Reply:
Yes, through the `sh:class` constraint in the specialised `SelectorShapes`, an `rdf:type` is already required. However, I also wanted to have a generic `SelectorShape` reusable by others and a `oa:Selector` requires a `rdf:type`.